### PR TITLE
Tag and build with spec

### PR DESCRIPTION
**Pull request by alvinypyim**
# Why?

This PR is really fulfilling the things described in the [README.md](https://github.com/sealink/docker-eb-deploy/blob/master/README.md) file.

### CODE REVIEW

**Comment by msmirnoff** [comment]
👍

File: bin/docker-release [hunk 1]
```diff
@@ -1,2 +1,18 @@
 #!/usr/bin/env ruby
-raise NotImplementedError
+
+THIS_FILE = File.symlink?(__FILE__) ? File.readlink(__FILE__) : __FILE__
```
Comment (alxberardi) [inline bin/docker-release]: You can actually make this a pathname object in ruby, which will simplify the use you make of it below

File: lib/docker/release/runner.rb [hunk 2]
```diff
@@ -0,0 +1,41 @@
+require 'deploy'
+require 'docker/repository'
+require 'docker/dockerrun/validate'
+
+module Docker
+  module Release
+    class Runner < ::Deploy::Runner
+      def run
+        trap_int
```
Comment (alxberardi) [inline lib/docker/release/runner.rb:9]: Where is this defined? In `::Deploy::Runner` ?

File: lib/docker/release/runner.rb [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+        precheck!
+        validate!
+        perform!
+      end
+
+      private
+
+      def precheck!
+        check_for_unstaged_changes!
```
Comment (alxberardi) [inline lib/docker/release/runner.rb:18]: I suppose these methods are defined in `::Deploy::Runner`

File: lib/docker/repository.rb [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+    end
+
+    def version!
+      super
+      tag_dockerrun!
+    end
+
+    def commit!
+      puts "Committing version.txt and #{Dockerrun::FILE_NAME}..."
+      unless system("git add public/version.txt #{Dockerrun::FILE_NAME}") && system("git commit -m \"#{commit_message}\" ")
```
Comment (alxberardi) [inline lib/docker/repository.rb:20]: I suppose that `commit_message` is defined in `::Deploy::Repository`

**Comment by alxberardi** [comment]
👍 minor comment, but please feel free to merge and release the gem

**Comment by mnoack** [comment]
:+1:

**Pull request merged**